tests/plugins/enterkey/manual/noselection.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.7.1, 16874
```

Comment:
The manual test only covers the issue case on Chrome (there is no selection when pressing enter) while in other browsers the selection exists and cannot be removed/dismissed by clicking inside of the editor.

Maybe it would be possible to add a second scenario for the test which will be covering button removing selection/ranges from the editor to check if `enter` does not produce an error then (similar as in unit test).

The exampe of test with more than one scenario: https://github.com/ckeditor/ckeditor-dev/blob/master/tests/plugins/clipboard/manual/13883.md.

Keep in mind that after pressing button outside of the editor, the editor loses focus - probably clicking should first focus the editor and then remove selection.